Resources/public/js/views/services/ez-contenteditviewservice.js

```diff
@@ -36,10 +36,12 @@ YUI.add('ez-contenteditviewservice', function (Y) {
          */
         _load: function (next) {
             var request = this.get('request'),
-                service = this;
+                service = this,
+                languageCode = request.params.languageCode;
 
+            this.set('languageCode', languageCode);
```

Comment:
this should not be done here, you should add an event handler on the `requestChange` event for that (and the corresponding test)